i3lock.c

```diff
@@ -709,7 +732,7 @@ static void handle_key_press(xcb_key_press_event_t *event) {
             case XKB_KEY_XF86AudioMute:
             case XKB_KEY_XF86AudioLowerVolume:
             case XKB_KEY_XF86AudioRaiseVolume:
-                xcb_send_event(conn, true, screen->root, XCB_EVENT_MASK_BUTTON_PRESS, (char *)event);
+                send_key_to_root(event, true);
```

Comment:
For some reason, Media and Volume keys didn't work when sending one key, so I had to send two.